internal/tfimport/importer/google_service_account_iam_binding.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package importer
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/terraform"
+)
+
+// ServiceAccountIAMBinding defines a struct with the necessary information for a google_service_account_iam_binding to be imported.
+type ServiceAccountIAMBinding struct{}
+
+// ImportID returns the ID of the resource to use in importing.
+func (b *ServiceAccountIAMBinding) ImportID(rc terraform.ResourceChange, pcv ProviderConfigMap) (string, error) {
```

Comment:
Do we individually need to support the other IAM types too like IAM member?

Reply:
Yes, they're imported differently. I'll add those to this PR too, it makes sense.

Reply:
Looks good, but I wonder if there are going to be way too many files in this dir. Maybe merge them to be more top level like google_iam, google_healthcare, google_storage, etc down the line>

Reply:
That's a good idea. I'd like to have all resources from FDA MyStudies working first, then I'll do a pass and combine into one file where appropriate.